config/prow/cluster/cherrypicker_deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
```

Comment:
I guess, you might want to add these manifests to `deploy.sh` :)

Reply:
😅